### PR TITLE
joseph smith added

### LINKED ISSUE #1502: [Add] joseph smith webdev
### Site URL

https://www.youtube.com/c/TheHelpingDevelop/

### Description of the above provided site

Learn programming and building with languages like HTML,CSS ,PHP and much more.

### Category

Youtube

### Sub Category

web development

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by S-ishita**
## Fixes Issue
closes #1502 

## Screenshots
Screenshot from 2023-08-10 18-24-04

### CODE REVIEW

**Comment by k-deepak04** [comment]
@S-ishita resolve the conflicts

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -326,5 +326,13 @@
     "category": "youtube",
     "subcategory": "web-development",
     "language": "english"
+  },
+  {
+    "name": "Joseph Smith",
+    "description": "Learn programming and building with languages like HTML,CSS ,PHP and much more.",
```
Comment (CBID2) [inline database/youtube/web-development.json:332]:
```suggestion
    "description": "Learn programming and building with languages like HTML,CSS ,PHP, and much more.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by rupali-codes — APPROVED** [review]
thanks, lgtm

**Pull request merged**